test/test_comment.py

```diff
@@ -0,0 +1,197 @@
+                    )
+                    self.assertEqual(
+                        inspect.signature(h).parameters["comment"].annotation, Union[Any, None]
+                    )
+                    if isinstance(maybe_cursor, CommandCursor):
+                        maybe_cursor.close()
+                    tested = False
+                    for i in results["started"]:
+                        if cc == i.command.get("comment", ""):
+                            tested = True
+                    self.assertTrue(
+                        tested,
+                        msg='Using the keyword argument "comment" did '
+                        "not work for func: %s with comment "
+                        "type: %s" % (h.__name__, type(cc)),
+                    )
+                    if h not in [coll.aggregate_raw_batches]:
+                        self.assertIn(
+                            "`comment` (optional):",
+                            h.__doc__,
+                            msg="Could not find 'comment' in the "
+                            "docstring of function %s" % (h.__name__),
+                        )
+                        if h not in already_supported:
+                            self.assertIn(
+                                "Added ``comment`` parameter",
+                                h.__doc__,
+                                msg="Could not find 'comment' "
+                                "versionchanged in "
+                                "the "
+                                "docstring of function %s" % (h.__name__),
+                            )
+                        else:
+                            self.assertNotIn(
+                                "Added ``comment`` parameter",
+                                h.__doc__,
+                                msg="Found 'comment' "
+                                "versionchanged in "
+                                "the "
+                                "docstring of function that "
+                                "should not have it %s" % (h.__name__),
+                            )
+
+        results.clear()
+
+    @client_context.require_version_min(4, 7, -1)
+    @client_context.require_replica_set
+    def test_database_helpers(self):
+        listener = EventListener()
+        db = rs_or_single_client(event_listeners=[listener]).db
+        helpers = [
+            (db.watch, []),
+            (db.command, ["hello"]),
+            (db.list_collections, []),
+            (db.list_collection_names, []),
+            (db.drop_collection, ["hello"]),
+            (db.validate_collection, ["test"]),
+            (db.dereference, [DBRef("collection", 1)]),
+        ]
+        already_supported = [db.command, db.list_collections, db.list_collection_names]
+        self._test_ops(helpers, already_supported, listener, db=db, coll=db.get_collection("test"))
+
+    @client_context.require_version_min(4, 7, -1)
+    @client_context.require_replica_set
```

Comment:
Is require_replica_set needed?

Reply:
Also for watch() I believe.

Reply:
Confirmed for both of them that `watch()` requires it.